tests/staking/ExtendedStakingTest.js

```diff
@@ -608,6 +608,34 @@ contract("Staking", (accounts) => {
             await expectRevert(staking.extendStakingDuration(lockedTS, newTime), "S04"); // S04 : cannot reduce staking duration
         });
 
+        it("Is protected from getting unlimited voting power by using corner case values in extendStakingDuration function", async () => {
+            let amount = "1337";
```

Comment:
Any reason why using this specific number?

Reply:
nah  
just copypasted from the vulnerability report  
changed for clarity though  
[commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/commit/1c02fa2a1b5af97f735a0a52d41ee9c1be8abcaf)